### PR TITLE
update analyze-runtime and timeline-tool devtools docs

### LINKED ISSUE #2663: document JS Profile and Paint better in timeline-tool
hi, i see the timeline doc. i hava some question:

1. how to use the two parts, it seems to not selected have difference between selected

 how_to_use_the_timeline_tool___web_tools_-_google_developers 2

2. there are many lines , i don't konw what it means

 how_to_use_the_timeline_tool___web_tools_-_google_developers

sorry, I'm a lot of problems recently. But I really can't find the answers to these questions, I really want to clear

### PR DESCRIPTION
**Pull request by kaycebasques**
@pbakaus please review

This is mainly updates to `analyze-runtime` with a couple additions to `timeline-tool`. Again, I think the perf docs could probably use a major overhaul, but this update is just to get `analyze-runtime` to a not-horribly-out-of-date status. 

fixes #2663

### CODE REVIEW

**Comment by petele** [comment]
@pbakaus - once you've reviewed and LGTM'ed, please remove the DO NOT MERGE tag.

File: src/content/en/tools/chrome-devtools/profile/evaluate-performance/timeline-tool.markdown [hunk 1]
```diff
@@ -127,6 +127,32 @@ reference][event reference] for details on each record type.
...
+Enable the **Paint** checkbox before you take a recording to gain more insight
+into **Paint** events. When paint profiling is enabled and you click
+on a **Paint** event, a new **Paint Profiler** tab is displayed in the 
+**Details** pane that shows much more granular information about the event.
+
+![paint profiler](imgs/paint-profiler.png)
+
+### Rendering settings {#rendering-settings}
+
+Open the main DevTools menu and select **More tools** > **Rendering settings**
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/profile/evaluate-performance/timeline-tool.markdown]: Does the main DevTools menu have another, perhaps better name? I guessed its location, but I'm not sure everyone can do that?

File: src/content/en/tools/chrome-devtools/profile/rendering-tools/analyze-runtime.markdown [hunk 2]
```diff
@@ -1,82 +1,73 @@
...
 
 ### Tools
 
-The simplest way to test JavaScript performance is to view Scripting events
-in the [Chrome DevTools Timeline](/web/tools/chrome-devtools/profile/evaluate-performance/timeline-tool).
-If any of the scripting events seem suspiciously long,
-enable the JS Stacks at the top of the Timeline tool:
-
-![JavaScript Profiler](imgs/js-stack.png)
+Make a **Timeline** [recording][recording] and look for suspiciously long
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/profile/rendering-tools/analyze-runtime.markdown]: Is there supposed to be a url in () here?

File: src/content/en/tools/chrome-devtools/profile/rendering-tools/analyze-runtime.markdown [hunk 3]
```diff
@@ -1,82 +1,73 @@
...
 
-The simplest way to test JavaScript performance is to view Scripting events
-in the [Chrome DevTools Timeline](/web/tools/chrome-devtools/profile/evaluate-performance/timeline-tool).
-If any of the scripting events seem suspiciously long,
-enable the JS Stacks at the top of the Timeline tool:
-
-![JavaScript Profiler](imgs/js-stack.png)
+Make a **Timeline** [recording][recording] and look for suspiciously long 
+**Evaluate Script** events. If you find any, you can enable the 
+[JS Profiler][profiler] and re-do your recording to get more detailed
```
Comment (jpmedley) [inline src/content/en/tools/chrome-devtools/profile/rendering-tools/analyze-runtime.markdown]: Same.

File: src/content/en/tools/chrome-devtools/profile/rendering-tools/analyze-runtime.markdown (hunk 2 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/profile/rendering-tools/analyze-runtime.markdown]: This is a reference-style link. See bottom of this section: https://daringfireball.net/projects/markdown/syntax#link

File: src/content/en/tools/chrome-devtools/profile/evaluate-performance/timeline-tool.markdown (hunk 1 above)
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/profile/evaluate-performance/timeline-tool.markdown]: Yah I had the same uneasiness in my gut. @pbakaus do we have a better name? Worst case, I could link to a section documenting how to access main menu...

Comment (pbakaus) [inline src/content/en/tools/chrome-devtools/profile/evaluate-performance/timeline-tool.markdown]: It's called "Main menu" everywhere, so we should stick with it for now. you could always embed the little three dot inline image, maybe..

**Comment by pbakaus** [comment]
LGTM.

**Pull request merged**